packages/components/src/theme/fileupload.scss

```diff
@@ -98,6 +98,12 @@
   }
 }
 
+.file-upload--is-disabled {
+    background-color: #EEEEEE;
+    color: #555555;
```

Comment:
we have scss variables for these right? `$gray-lighter` and `$gray`

Reply:
Good call. Fixed.